test/controllers/api/v1/developments_controller_test.rb

```diff
@@ -11,19 +11,25 @@ def development
   end
 
   test 'should get index, filtering on range' do
-    get :index, filter: { commsf: '[11,13]' }
-    assert_equal 1, results(response).count
+    get :index, filter: { commsf: "[11,13]" }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.